ext/XS-APItest/t/call.t

```diff
@@ -340,6 +340,16 @@ for my $fn_type (qw(eval_pv eval_sv call_sv)) {
     }
 }
 
+{
+    use feature "fc";
+    # the XS eval_sv() returns the count of results
+    is(eval_sv('fc("A") eq fc("a"); 1', G_LIST), 0,
+       "don't inherit hints by default (so the eval fails)");
+    is(eval_sv('fc("A") eq fc("a"); 1', G_LIST | G_USEHINTS), 1,
+       "inherit hints when requested (so the eval succeeds)")
+      or diag($@);
+}
+
```

Comment:
Possibly also wants a test that the `strict` flags get inherited. Maybe check some code like `5 + $notavariable` and check it's 5 without the hints but doesn't even compile with.